config/routes.rb

```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
+  root to: 'keyword#index'
+  get 'keyword/index'
```

Comment:
can we use `resources :keywords, only: :index`

Reply:
Yes thanks!
I've also renamed keywordController into keyword**_s_**Controller, it seems more natural as their will be many keywords.